distribution/src/resources/config-tool/deployment-full.toml

```diff
@@ -632,14 +650,18 @@ sender.truststore_password = "$ref{truststore.password}" # inferred
 ###################################     SAP configurations          ###################################
 [transport.sap] # optional
 
-listener.enable = true
+listener.enable = true                      # enable idoc by default
+listener.idoc.enable = true                 # idoc enable by default if transport.sap.listener.enable=true
```

Comment:
Let's not enable idoc by default. IDoc and BAPI are 2 different protocols which need to be specified explicitely. Setting 1 by default is not correct.